pkgs/development/tools/misc/gnum4/default.nix

```diff
@@ -13,7 +13,14 @@ stdenv.mkDerivation {
   configureFlags = [ "--with-syscmd-shell=${stdenv.shell}" ];
 
   # Upstream is aware of it; it may be in the next release.
-  patches = [ ./s_isdir.patch ]
+  patches =
+    [
+      ./s_isdir.patch
+      (fetchurl {
```

Comment:
fetchpatch?

Reply:
This isn't a dynamically generated patch, just simple http(s) download.